lib/DataSift/ApiClient.php

```diff
@@ -67,7 +67,12 @@ static public function call(
         $ssl = $user->useSSL();
 
         // Build the full endpoint URL
-        $url = 'http'.($ssl ? 's' : '').'://'.$user->getApiUrl(). $user->getApiVersion() . '/'. $endPoint;
+        if ($method == 'ingest') {
```

Comment:
Surely we could pass in a flag on the call? If it's flagged as ingestion we swap the url on initialise? Then we don't need these extra additions to the methods, which I've flagged below?